data/resources/ui/content-message-sticker.ui

```diff
@@ -5,7 +5,21 @@
       <object class="GtkBinLayout"/>
```

Comment:
```suggestion
      <object class="GtkBoxLayout"/>
```

This way you don't need the GtkBox below. You also need to add the orientation and spacing properties to it.